.github/workflows/build_mpv_binary_and_deb_package.yml

```diff
@@ -0,0 +1,139 @@
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y \
+            build-essential \
+            clang \
+            yasm \
+            git \
+            meson \
+            ninja-build \
+            autoconf \
+            automake \
+            libtool \
+            pkg-config \
+            libfreetype6-dev \
+            libfribidi-dev \
+            libfontconfig1-dev \
+            libasound2-dev \
+            pulseaudio \
+            libx11-dev \
+            libxss-dev \
+            libvdpau-dev \
+            libgl-dev \
+            libxv-dev \
+            libjpeg-dev \
+            libssl-dev \
+            libx264-dev \
+            libmp3lame-dev \
+            libfdk-aac-dev \
+            python3-pip \
+            python3.12-venv \
+            libharfbuzz-dev \
+            nasm \
+            liblua5.1-0-dev \
+            liblua5.2-dev \
+            libxext-dev \
+            libxrandr-dev \
+            libxinerama-dev \
+            libxv-dev \
+            libwayland-dev \
+            nodejs \
+            libmujs-dev \
+            liblcms2-dev \
+            libarchive-dev \
+            libbluray-dev \
+            libavdevice-dev \
+            cmake \
+            libuchardet-dev \
+            libjack-dev \
+            libopenal-dev \
+            libpipewire-0.3-dev \
+            libpulse-dev \
+            libsndio-dev \
+            libcaca-dev \
+            libdrm-dev \
+            libsixel-dev \
+            wayland-protocols \
+            libxkbcommon-dev \
+            libxpresent-dev \
+            libegl1-mesa-dev \
+            libvulkan-dev \
+            libva-dev \
+            libshaderc-dev \
+            && sudo apt-get clean
+
+    - name: Build mpv binary
+      run: |
+        ./rebuild -j4
+
+    - name: Install mpv binary
+      run: |
+        sudo ./install
+
+    - name: Verify mpv binary
+      run: |
+        mpv --version
+
+    - name: Create debian package
+      run: |
+        # enable source repositories
+        sudo sed -i 's/^Types: deb/Types: deb deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
+        sudo apt-get update
+        # install dependencies mentioned in debian/control
+        sudo apt-get install -y \
+            ladspa-sdk \
+            libbs2b-dev \
+            libdisplay-info-dev \
+            libdav1d-dev \
+            libepoxy-dev \
+            libgl1-mesa-dev \
+            libgnutls28-dev \
+            libmodplug-dev \
+            libopus-dev \
+            libopencore-amrnb-dev \
+            libopencore-amrwb-dev \
+            librtmp-dev \
+            libssh-dev \
+            libsoxr-dev \
+            libspeex-dev \
+            libv4l-dev \
+            libvorbis-dev \
+            libvo-amrwbenc-dev \
+            libunwind-dev \
+            libvpx-dev \
+            libxvidcore-dev \
+            && sudo apt-get clean
+
+        # build dependencies
+        sudo apt-get build-dep mpv
```

Comment:
this *installs* build dependencies but I am pretty sure this references whatever dependencies the `mpv` repo package has and not what's inside `debian/control`